_posts/2017-10-23-pair-programming-with-perry.md

```diff
@@ -0,0 +1,91 @@
+</div>
+<br>
+<span class='os-green'>*That’s easy, especially for me because I’m wicked smart.*</span>
+<div style='margin-left: 2em;' >
+    That’s the spirit. I also place a lot of value on the conversations between you and your pair. The exchanges are what make pairing so valuable. This goes back to what I first talked about regarding self-explanations and rubber duck programming. Pairing allows you to continuously get feedback, potentially triggering those ‘ah-ha’ moments more frequently. Instead of working solo and making decisions on your own you can benefit from having another engineer right next to you to collaborate on code design.
+    <br><br>
+    This leads me to a few items I forgot to mention, which are effortless knowledge transfer and additional code comprehension. When you pair with someone you’ll inevitably learn things from each other. Now these can be small things like keyboard shortcuts or useful built in functions of a language, but there is also a high likelihood of learning about a section of code you haven’t had experience with before.
+    <br><br>
+    As far as code comprehension goes, pairing results in two people having a clear understanding of the code that was just written as opposed to the usual one. This is more of a long-term gain because there is always the possibility that the person that worked on the code will not be available to help explain things, so having an additional person with that same knowledge could pay off big down the road.
+</div>
+<br>
+<span class='os-green'>*Alright, Perry, you seem to really advocate pair programming. Are there any cons to pairing?*</span>
+<div style='margin-left: 2em;' >
+    There definitely are, though I think the good outweigh the bad by a long shot.
+    <br><br>
+    Pairing forces you to sit next to someone and interact, and some days you don’t want to; you’d much rather fly solo at your desk with your headphones on, which is totally fine, but the forced interaction can be seen as a drawback.
+    <br><br>
+    The possibility is always there that you’ll butt heads with your pair, which isn’t fun for anybody. However, I think this offers a chance to improve interpersonal skills as you’ll learn to compromise and communicate more effectively.
+    One last negative I’ll note is that some believe pairing reduces your team’s velocity. This isn’t something I agree with. I believe pairing pays short term and long term dividends with all of the pros I talked about before.
+</div>
+<br>
+<span class='os-green'>*That’s something that came to my mind right away is that ‘I have to sit and talk to somebody all day?!’ Doesn’t sound so fun to me.*</span>
```

Comment:
*"That's something that came to my mind right away: 'I have to sit and talk to somebody all day?!'"